products/ubuntu.md

```diff
@@ -17,8 +17,8 @@ auto:
     template: "{{v1}}{%if v2%}\n{{v2}}{%endif%}"
 identifiers:
 -   purl: pkg:os/ubuntu
-activeSupportColumn: Hardware and Maintenance Updates
-eol: Maintenance Updates
+activeSupportColumn: Hardware & Maintenance
+eol: Maintenance/Security Support
```

Comment:
Should be `eolColumn` ?

And the label should be just `Maintenance` to match with the description (with `updates` omitted like for `activeSupportColumn`).

Reply:
Maintenance is somewhat unclear, so want a mention of "Security"

Reply:
If you think of a smaller label it would prevent the table to scroll horizontally.

I did not found one with "security" in it though, and I still think the following makes sense :

1. Hardware & Maintenance
2. Maintenance
3. Extended Security Maintenance

Reply:
> If you think of a smaller label it would prevent the table to scroll horizontally.

Maybe `Maintenance & Security Support` ?